backend/kernelCI_app/helpers/filters.py

```diff
@@ -404,15 +404,15 @@ def _handle_platforms(self, current_filter: ParsedFilter) -> None:
         tab = current_filter["field"].split(".")[0]
         self.filterPlatforms[tab].add(current_filter["value"])
 
-    def _processFilters(self):
+    def _process_filters(self):
         try:
             for current_filter in self.filters:
                 field = current_filter["field"]
                 # Delegate to the appropriate handler based on the field
                 if field in self.filter_handlers:
                     self.filter_handlers[field](current_filter)
-        except InvalidComparisonOP as e:
-            return HttpResponseBadRequest(getErrorResponseBody(str(e)))
+        except InvalidComparisonOPError as e:
+            return HttpResponseBadRequest(get_error_body_response(str(e)))
```

Comment:
Do we use this response structure actively? If not, I'd argue it's safe to make this change
```suggestion
            return create_error_response(error_message=str(e), status_code=HTTPStatus.BAD_REQUEST)
```

or using `create_api_error_response`. Maybe not for this PR, but definitely something we could do